src/services/getfilteredTodos.ts

```diff
@@ -0,0 +1,33 @@
+import { Todo } from '../types/Todo';
+import { FilterParams } from '../types/filterParams';
+
+export function getFilteredTodos(
+  todos: Todo[],
+  filterParams: FilterParams,
+  title?: string,
+) {
+  let filteredTodos = [...todos];
+
+  if (filterParams !== FilterParams.All) {
+    filteredTodos = filteredTodos.filter(todo => {
+      switch (filterParams) {
+        case FilterParams.Active:
+          return !todo.completed;
+
+        case FilterParams.Completed:
+          return todo.completed;
+
+        default:
+          return todo;
```

Comment:
```suggestion
          return true;
```
the return value of filter callback must be a boolean